cli/common.go

```diff
@@ -36,10 +36,21 @@ func fillCommonRuntime() (*torcx.CommonConfig, error) {
 		ConfDir: torcx.DefaultConfDir,
 		StorePaths: []string{
 			torcx.VendorStoreDir,
-			torcx.OemStoreDir,
 		},
 	}
 
+	// Determine OS release ID
+	OsRelease, err := torcx.CurrentOsVersionID(torcx.OsReleasePath)
+	if err != nil {
```

Comment:
Probably warn. In what situation could this fail? I can't immediately think of one.